### ISSUE TITLE
Window scaling doesn't apply to textboxes

### ISSUE THREAD

**Issue by wonea** [Acolarh/colorgrab#8]
On Windows 10 with screen scaling set to 125% it doesn't resize textboxes correctly.

![image](https://user-images.githubusercontent.com/557192/36431157-c5a2d762-164e-11e8-8e6a-8b4a1cb3f354.png)

**Comment by nielssp** [comment]
Hm, just tried it with 125% scaling on Windows 10 and got the following:
![cgwin10](https://user-images.githubusercontent.com/424372/36433782-aa6e373e-165d-11e8-8e77-96c16fc65baf.png)
However, I did notice that with 125% scaling the color picker didn't work correctly; the picked color did not match the color under the cursor. It seemed like the cursor offset was somehow being scaled.

I also tried 125% scaling on Windows 7 which looked more like your screenshot, i.e. only the text was scaled and didn't fit in the text boxes.
